setup.cfg

```diff
@@ -37,4 +37,6 @@ console_scripts =
 
 [flake8]
 max-line-length = 120
```

Comment:
Black's default is 88. And most docstrings are formatted to comply with that line length.

Reply:
Black's suggested config for flake8
https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#minimal-configuration